src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Constants.scala

```diff
@@ -6,6 +6,8 @@ object Constants {
   val SparkStreamingBatchSizeDefault = 5
 
   val EventHubProgressDir = "eventhubProgress"
+  val ContextStopWaitTimeMillis = 5000
```

Comment:
I think I need to change this to allow for more wiggle room in cases where the user is adding more than one stream. Maybe increase this to 1 minute?